storagenode/console/consoleserver/server.go

```diff
@@ -49,18 +48,18 @@ type DashboardResponse struct {
 
 // DashboardData stores all needed information about storagenode
 type DashboardData struct {
-	Bandwidth          console.BandwidthInfo       `json:"bandwidth"`
-	DiskSpace          console.DiskSpaceInfo       `json:"diskSpace"`
-	WalletAddress      string                      `json:"walletAddress"`
-	VersionInfo        version.Info                `json:"versionInfo"`
-	IsLastVersion      bool                        `json:"isLastVersion"`
-	Uptime             time.Duration               `json:"uptime"`
-	NodeID             string                      `json:"nodeId"`
-	Satellites         storj.NodeIDList            `json:"satellites"`
-	UptimeCheck        nodestats.ReputationStats   `json:"uptimeCheck"`
-	AuditCheck         nodestats.ReputationStats   `json:"auditCheck"`
-	BandwidthChartData []console.BandwidthUsed     `json:"bandwidthChartData"`
-	DiskSpaceChartData []nodestats.SpaceUsageStamp `json:"diskSpaceChartData"`
+	Bandwidth          console.BandwidthInfo    `json:"bandwidth"`
+	DiskSpace          console.DiskSpaceInfo    `json:"diskSpace"`
+	WalletAddress      string                   `json:"walletAddress"`
+	VersionInfo        version.Info             `json:"versionInfo"`
+	IsLastVersion      bool                     `json:"isLastVersion"`
+	Uptime             time.Duration            `json:"uptime"`
+	NodeID             string                   `json:"nodeId"`
+	Satellites         storj.NodeIDList         `json:"satellites"`
+	UptimeCheck        console.ReputationStats  `json:"uptimeCheck"`
+	AuditCheck         console.ReputationStats  `json:"auditCheck"`
```

Comment:
`console` package seems to be accumulating concerns from all the things.

Reply:
I agree, but may be it's better to fix already pushed `console` code in separate PR